src/Neo/ProtocolSettings.cs

```diff
@@ -58,12 +58,18 @@ public record ProtocolSettings
         public string[] SeedList { get; init; }
 
         /// <summary>
-        /// Indicates the time in milliseconds between two blocks.
+        /// Indicates the time in milliseconds between two blocks. Note that starting from
+        /// HF_Echidna block generation time is managed by native Policy contract, hence
+        /// use NeoSystemExtensions.GetTimePerBlock extension method instead of direct access
+        /// to this property.
         /// </summary>
         public uint MillisecondsPerBlock { get; init; }
```

Comment:
Rpc still using this variable, isn't it?

Reply:
perhaps it needs to be updated as well

Reply:
Ping @AnnaShaleva

Reply:
RPC still will use it, and it's still a part of the `getversion` response. I don't think we need to remove/adjust it in any way, because it's still a part of the node's protocol.

Reply:
So @shargon, I checked all the usages of `MillisecondsPerBlock` in both RPC server and RPC client, and to me, there's nothing to adjust. Could you please clarify, what should be adjusted?

Reply:
Although check out the ea04f2e619834a448cf5f82debbe46c9a5cb0784.